src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/Implementation/AWSLambdaUtils.cs

```diff
@@ -100,6 +151,35 @@ internal static string GetAccountId(string functionArn)
             return null;
         }
 
+        private static string GetFaasId(string functionArn)
+        {
+            var faasId = functionArn;
+
+            // According to faas.id description https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/instrumentation/aws-lambda.md#all-triggers
+            // the 8th part of arn (version, see https://docs.aws.amazon.com/lambda/latest/dg/configuration-versions.html#versioning-versions-using)
```

Comment:
The 8th part may also be an alias, which is actually the main reason for removing it. Maybe this link would be better since it links to both the Lambda version and alias docs while also providing a short description of the ARN format: https://docs.aws.amazon.com/lambda/latest/dg/lambda-api-permissions-ref.html

Reply:
Thanks. I used this link.